scripts/dumps/lib/serialize_entity_in_turtle.coffee

```diff
@@ -2,13 +2,15 @@ CONFIG = require 'config'
 __ = CONFIG.universalPath
 _ = __.require 'builders', 'utils'
 { properties } = __.require 'controllers', 'entities/lib/properties'
+{ yellow } = require 'chalk'
 
 module.exports = (entity)->
   { _id } = entity
 
   text = "inv:#{_id} a wikibase:Item ;"
 
   for lang, value of entity.labels
+    value = formatStringValue value
     text += """\n  rdfs:label #{formatStringValue(value)}@#{lang} ;"""
```

Comment:
`formattedValue` instead of `value` ? which can then be used as : 
```
text += """\n  rdfs:label #{formattedValue}@#{lang} ;"""
```
and instead of formattedValue which is very generic why not using `slug`

Reply:
isn't `slug` specific to URLs?

Reply:
`label` then ? something more specific than value?

Reply:
addressed in c7cc70cb